index.js

```diff
@@ -217,26 +219,25 @@ var appServer = function(config) {
               cert: certificate
             };
 
-            if (config.passphrase != undefined) {
-              credentials.passphrase = config.passphrase
+            if (self.config.passphrase != undefined) {
+              credentials.passphrase = self.config.passphrase
             }
 
-            if (chain != undefined) { //if chain is used the add to credentials
+            if (chain != undefined) {
               credentials.ca = chain;
               self.log("using chain certificate from /sslcert");
             }
 
-            try { // These two lines below can fail it the certs were generated incorrectly. But we can continue startup without HTTPS
-              var httpsServer = https.createServer(credentials, self.express); // create the HTTPS server
+            try {
+              // this can fail it the certs were generated incorrectly
+              var httpsServer = https.createServer(credentials, self.express);
 
-              // TODO: add separate option to specify specific host address for HTTPS server to bind to???
-              // Issue #38: https://github.com/alexa-js/alexa-app-server/issues/38
               if (typeof config.host === 'string') {
-                self.httpsInstance = httpsServer.listen(config.httpsPort, config.host);
-                self.log("listening on https://" + config.host + ":" + config.httpsPort);
+                self.httpsInstance = httpsServer.listen(self.config.httpsPort, self.config.host);
+                self.log("listening on https:// " + self.config.host + ":" + self.config.httpsPort);
```

Comment:
Might want to remove the space from `"https://"` so that if anyone is copying the exact url, it shouldn't be the following: `"https:// 127.0.0.1:6000"`